packages/cli/src/index.js

```diff
@@ -1,10 +1,23 @@
-const fs = require('fs');
 const getContainerStats = require('./getContainerStats');
 const onStartContainer = require('./onStartContainer');
+const exec = require('./exec');
+const computeMeasureAverage = require('./computeMeasureAverage');
 
 const run = async () => {
     const measureName = process.env.NAME;
-    onStartContainer(getContainerStats(measureName));
+    const command = process.env.COMMAND;
+    const runQuantity = process.env.RUN_QUANTITY;
+
+    for (var i = 1; i <= runQuantity; i++) {
```

Comment:
```suggestion
    for (let i = 1; i <= runQuantity; i++) {
```

Reply:
Using let here is actually a bad idea. Due to let scoping, a new variable will get created on every loop. I once had a bad memory leak due to this. One that turned into a segfault.